app/src/main/java/mozilla/lockbox/model/ToastNotificationViewModel.kt

```diff
@@ -0,0 +1,15 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package mozilla.lockbox.model
+
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+
+data class ToastNotificationViewModel(
+    @StringRes val strId: Int,
+    @DrawableRes val img: Int
```

Comment:
I would definitely add a

```kotlin
val untranslatableString: String? = null
```

here. Most of the time it can default to `null` but occasionally— e.g. `ShowDeleteToast` we'll need it to insert into the message.